From ad3490e9908ce0a65ef40576520fb33236b5c102 Mon Sep 17 00:00:00 2001
From: Dominik Csapak <d.csapak@proxmox.com>
Date: Tue, 13 Dec 2016 16:03:41 +0100
Subject: [PATCH 04/10] add pve style

this adds the custom pve style (based on black.css)

we hide the connect button, and add custom colors,
and fix the z-index of the connect overlay

Signed-off-by: Dominik Csapak <d.csapak@proxmox.com>
---
 app/styles/pve.css | 42 ++++++++++++++++++++++++++++++++++++++++++
 vnc.html           |  1 +
 2 files changed, 43 insertions(+)
 create mode 100644 app/styles/pve.css

diff --git a/app/styles/pve.css b/app/styles/pve.css
new file mode 100644
index 0000000..eaeb5cb
--- /dev/null
+++ b/app/styles/pve.css
@@ -0,0 +1,42 @@
+/*
+ * noVNC black CSS
+ * Copyright (C) 2012 Joel Martin
+ * Copyright (C) 2013 Samuel Mannehed for Cendio AB
+ * noVNC is licensed under the MPL 2.0 (see LICENSE.txt)
+ * This file is licensed under the 2-Clause BSD license (see LICENSE.txt).
+ */
+
+.noVNC_panel {
+    border: 0px;
+    background:#4c4c4c;;
+    color:#fff;
+}
+
+#noVNC_control_bar, #noVNC_control_bar_handle, .noVNC_panel .noVNC_heading {
+    background: #4c4c4c;
+}
+
+.noVNC_button.noVNC_selected {
+    background: #3892d4;
+}
+
+#pve_commands > input[type=button] {
+    width: 100%;
+}
+
+#noVNC_canvas:focus {
+    outline: none;
+}
+
+#noVNC_transition {
+    z-index: 0;
+}
+
+#noVNC_connect_button {
+    display: none;
+}
+
+:root:not(.noVNC_touch) .noVNC_button.noVNC_selected:hover {
+  border-color: rgba(0, 0, 0, 0.4);
+  background: #5BA8DF;
+}
diff --git a/vnc.html b/vnc.html
index 6c041b5..3110e8b 100644
--- a/vnc.html
+++ b/vnc.html
@@ -55,6 +55,7 @@
 
     <!-- Stylesheets -->
     <link rel="stylesheet" href="app/styles/base.css" />
+    <link rel="stylesheet" href="/novnc/app/styles/pve.css" />
 
     <!--
     <script type='text/javascript'
-- 
2.11.0

